css/style.css

```diff
@@ -0,0 +1,2921 @@
+}
+
+html {
+  box-sizing: border-box;
+}
+
+*, *::before, *::after {
+  box-sizing: inherit;
+}
+
+body {
+  font-family: 'Montserrat', sans-serif;
+  font-style: normal;
+  font-size: 15px;
+  line-height: 1.2;
+  font-weight: normal;
+  color: #000; 
+  margin: 0;
+}
+
+.container {
+  max-width: 1440px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 24px;
+  margin-right: 15px;
+}
+
+.header__logo {
+  display: inline-flex;
+  background-color: #fff;
+  padding-left: 42px;
+}
+
+.header__logo-img {
+  background-color: #fff;
+}
+
+.menu {
+  margin-left: 73px;
+}
+
+.menu__link {
+  font-weight: 600;
+  font-size: 15px;
+  line-height: 18px;
+  padding: 6px;
+  margin-right: 40px;
+  color: #000000;
+  border-bottom: 1px solid rgba(229, 25, 55, 0);
+  text-decoration: none;
+}
+
+.menu__link:nth-child(1) {
+  margin-left: 100px;
+  margin-right: 60px;
+  padding-right: 0;
+  padding-left: 0;
+}
+
+.menu__link:nth-child(2) {
+  padding-left: 0;
+  padding-right: 0;
+  margin-right: 18px;
+  margin-left: 0;
+}
+
+.menu__link:nth-child(3) {
+  padding-left: 0;
+  padding-right: 0;
+  margin-right: 20px;
+  margin-left: 4px;
+}
+
+.menu__link:nth-child(4) {
+  padding-right: 0;
+  padding-left: 0;
+  margin-right: 30px;
+  margin-left: 20px;
+}
+
+.menu__link:nth-child(5) {
+  padding-right: 0;
+  padding-left: 0;
+  margin-right: 98px;
+  margin-left: 12px;
+}
+
+.menu__link:hover, .menu__link:focus, .menu__link:active {
+  color: #e51937;
+  border-bottom: 2px solid #e51937;
+}
+
+.header__right {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  flex-flow: row nowrap;
+  margin-right: 45px;
+  text-decoration: none;
+}
+
+.header__menu {
+  margin-left: 33px;
+  padding-right: 0;
+  padding-left: 0;
+}
+
+.header__menu-link {
+  font-weight: 600;
+  font-size: 15px;
+  line-height: 18px;
+  margin-right: 30px;
+  color: #000000;
+  padding-left: 0;
+  margin-right: 20px;
+  margin-left: 8px;
+  text-decoration: none;
+}
+
+.header__menu-link:nth-child(2) {
+  padding-left: 0;
+  padding-right: 0;
+  margin-right: 25px;
+}
+
+.header__menu-link:hover, .header__menu-link:focus, .header__menu-link:active {
+  width: 60px;
+  height: 2px;
+  color: #e51937;
+  border-bottom: 2px solid #e51937;
+}
+
+.header__burger {
+  padding-right: 0;
+  padding-top: 3px;
+  padding-left: 3px;
+}
+
+.top {
+  text-align: center;
+  background: url(../img/tesla-fon.png)  no-repeat;
+  background-position: 50% 0;
+  background-size: 100%;
+}
+
+.top__title {
+  max-height: 100px;
+  margin-top: 60px;
+  font-weight: 600;
+  font-size: 90px;
+  margin-bottom: 0;
+}
+
+.top__subtitle {
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 18px;
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+.top__img {
+  max-width: 100%;
+  height: 450px;
+  margin-top: 25px;
+  margin-right: 75px; 
+}
+
+.top__wrapper {
+  display: flex;
+  justify-content: center;
+  margin-top: 10px; 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/vovafinal/pullRequest?prid=5998857)